plugins/cisco_umbrella_enforcement/komand_cisco_umbrella_enforcement/actions/delete_domain_by_id/action.py

```diff
@@ -14,16 +14,8 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        ID = params.get("ID")
+        ID = params.get(Input.ID)
 
-        try:
-            status_code = self.connection.api.delete_domains_by_id(ID)
-            self.logger.info(status_code)
-        except Exception:
-            self.logger.error("DeleteDomainById: run: Problem with request")
-            raise Exception("DeleteDomainById: run: Problem with request")
+        result = self.connection.client.delete_event(domain_id=ID)
 
-        return {"status": "success"}
-
-    def test(self):
-        return {"status": "success"}
+        return {Output.STATUS: result}
```

Comment:
Should all these test functions be removed? (Test only required for connection? )

Reply:
Test is only required for connection :)

Reply:
Thanks for confirming, can you remove the test functions out of all the dev files (other than the connection one)?